composer.json

```diff
@@ -13,10 +13,10 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zend-config": "2.2.3",
-        "zendframework/zend-di": "2.2.3"
+        "zendframework/zend-config": "~2.3.7",
```

Comment:
this may potentially break things for people. We do not require 2.3.7, although we support it.
Can you please change the version constraints to be ~2.3, if you think it makes sense?